### PR TITLE
Fixed IFrames of Billing SCA pages

### PR DESCRIPTION
**Pull request by jsmolar**

### CODE REVIEW

File: testsuite/ui/browser.py [hunk 1]
```diff
@@ -104,3 +107,24 @@ def new_tab(self, trigger, keep_tab=False):
         else:
             self.browser.close_window(new_handle)
             self.browser.switch_to_window(current_handle)
+
+    @backoff.on_exception(backoff.fibo, NoSuchElementException, max_tries=13, jitter=None)
```
Comment (JaurbanRH) [inline testsuite/ui/browser.py:111]: I think max_tries=13 is too much. Use e.g. 8 as in rest of testsuite.

Comment (jsmolar) [reply testsuite/ui/browser.py:111]: 8 was sometimes not enough for the OTP form to load.

**Review by dhlavac — APPROVED** [review]

File: testsuite/ui/views/devel/settings/stripe.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 
 
-class BillingAddressForm(View):
+class _BillingAddressForm(View):
```
Comment (pehala) [inline testsuite/ui/views/devel/settings/stripe.py:14]: I dont think this name change is needed as it requires module-level pylint ignore.

File: testsuite/ui/views/devel/settings/stripe.py [hunk 3]
```diff
@@ -35,50 +35,57 @@ def add(self, address):
...
+
+class _StripeCCForm(View):
     """Stripe Credit card form"""
-    submit_btn = GenericLocatorWidget("//button[@id='stripe-submit']")
+    submit_btn = GenericLocatorWidget(".//button[@id='stripe-submit']")
+
+    @View.nested
+    # pylint: disable=invalid-name
+    class cc_details(View):
+        """I frame that contains credit card information"""
```
Comment (pehala) [inline testsuite/ui/views/devel/settings/stripe.py:50]: IFrame

File: testsuite/ui/views/devel/settings/stripe.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 """Settings Devel portal View containing credit card details for Stripe payment gateway"""
+# pylint: disable=invalid-name
```
Comment (mganisin) [inline testsuite/ui/views/devel/settings/stripe.py:2]: ???

File: testsuite/ui/views/devel/settings/stripe.py [hunk 5]
```diff
@@ -35,50 +35,57 @@ def add(self, address):
...
 
-class StripeCCForm(View):
+
+class _StripeCCForm(View):
     """Stripe Credit card form"""
-    submit_btn = GenericLocatorWidget("//button[@id='stripe-submit']")
+    submit_btn = GenericLocatorWidget(".//button[@id='stripe-submit']")
+
+    @View.nested
+    # pylint: disable=invalid-name
```
Comment (mganisin) [inline testsuite/ui/views/devel/settings/stripe.py:48]: is this necessary?

File: testsuite/ui/views/devel/settings/stripe.py (hunk 2 above)
Comment (mganisin) [reply testsuite/ui/views/devel/settings/stripe.py:14]: Good question. Why? (though it is valid name in my opinion)

File: testsuite/ui/views/devel/settings/stripe.py [hunk 6]
```diff
@@ -35,50 +35,57 @@ def add(self, address):
...
+    class cc_details(View):
+        """I frame that contains credit card information"""
+        FRAME = ".//form[@id='stripe-form']//iframe"
+        cc_number = TextInput(name="cardnumber")
```
Comment (mganisin) [inline testsuite/ui/views/devel/settings/stripe.py:52]: **cardnumber** = TextInput(name="**cardnumber**")

File: testsuite/ui/views/devel/settings/stripe.py (hunk 2 above)
Comment (jsmolar) [reply testsuite/ui/views/devel/settings/stripe.py:14]: This one does not require pylint ignore. 
`BillingAddressFroms` is used only by `StripeCCForm` so I assumed that package-private would be correct encapsulation

File: testsuite/ui/views/devel/settings/stripe.py (hunk 4 above)
Comment (jsmolar) [reply testsuite/ui/views/devel/settings/stripe.py:2]: What is more preferable? To use module-level pylint ignore or use the same one on 3 different places in a module?

Comment (pehala) [reply testsuite/ui/views/devel/settings/stripe.py:2]: Definitely 3 in the exact spots they are needed

**Review by mganisin — APPROVED** [review]

**Review by JaurbanRH — APPROVED** [review]

**Pull request merged**